src/main.tf

```diff
@@ -0,0 +1,4 @@
+module "memcached" {
+  source = "../../terraform-modules/massdriver/aws-elasticache-memcached"
+  md_metadata = var.md_metadata
```

Comment:
Pass non-md specifics into cloud modules...

ie:
name = var.md_metadata.name_prefix
tags = var.md_metadta.default_tags 

We want to keep these pure so we can publish them on the TF reg.

Reply:
Ahh! I thought I finally got it [here](https://github.com/massdriver-cloud/terraform-modules/pull/98), but one more adjustment.

The thinking here was

```
md_metadata = md_metadata
# OR
name = var. md_metadata.name_prefix
tags = var.md_metadata.default_tags
```

Which after typing that out, having 2 instead of one isn't so bad and that interface is nicer.... Thanks for clearing this up once and for all.